### ISSUE TITLE
Unable to rotate screen via mod+r

### ISSUE THREAD

**Issue by HiramTheHero** [Genymobile/scrcpy#4839]
- [YES] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [YES] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: macOS M1
 - scrcpy version: scrcpy 2.4
 - installation method: brew
 - device model: Pixel 5a
 - Android version: 14

**Describe the bug**
When pressing `mod+r` to rotate the screen, I get the following error:

```
[server] INFO: Device rotation requested: landscape
[server] ERROR: Could not invoke method
java.lang.NoSuchMethodException: android.view.IWindowManager$Stub$Proxy.freezeRotation [int]
	at java.lang.Class.getMethod(Class.java:2937)
	at java.lang.Class.getMethod(Class.java:2449)
	at com.genymobile.scrcpy.wrappers.WindowManager.getFreezeRotationMethod(WindowManager.java:48)
	at com.genymobile.scrcpy.wrappers.WindowManager.freezeRotation(WindowManager.java:111)
	at com.genymobile.scrcpy.Device.rotateDevice(Device.java:372)
	at com.genymobile.scrcpy.Controller.handleEvent(Controller.java:205)
	at com.genymobile.scrcpy.Controller.control(Controller.java:97)
	at com.genymobile.scrcpy.Controller.lambda$start$0$com-genymobile-scrcpy-Controller(Controller.java:105)
	at com.genymobile.scrcpy.Controller$$ExternalSyntheticLambda1.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:1012)
```

Prior screen orientation is irrelevant.

```
[server] INFO: Device rotation requested: portrait
[server] ERROR: Could not invoke method
java.lang.NoSuchMethodException: android.view.IWindowManager$Stub$Proxy.freezeRotation [int]
	at java.lang.Class.getMethod(Class.java:2937)
	at java.lang.Class.getMethod(Class.java:2449)
	at com.genymobile.scrcpy.wrappers.WindowManager.getFreezeRotationMethod(WindowManager.java:48)
	at com.genymobile.scrcpy.wrappers.WindowManager.freezeRotation(WindowManager.java:111)
	at com.genymobile.scrcpy.Device.rotateDevice(Device.java:372)
	at com.genymobile.scrcpy.Controller.handleEvent(Controller.java:205)
	at com.genymobile.scrcpy.Controller.control(Controller.java:97)
	at com.genymobile.scrcpy.Controller.lambda$start$0$com-genymobile-scrcpy-Controller(Controller.java:105)
	at com.genymobile.scrcpy.Controller$$ExternalSyntheticLambda1.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by rom1v** [comment]
Please test `dev` branch, it should be fixed (by #4740).

**Comment by HiramTheHero** [comment]
Works on `dev` branch. Thank you!
